marble_client/client.py

```diff
@@ -51,6 +69,38 @@ def __init__(self, fallback: Optional[bool] = True) -> None:
     def nodes(self) -> dict[str, MarbleNode]:
         return self._nodes
 
+    @property
+    @cache
+    @check_jupyterlab
+    def current_node(self) -> MarbleNode:
```

Comment:
Maybe we rename this to `this_node`?

Reply:
Yes good idea, I remember we talked about this